code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -249,77 +301,68 @@
 			if(world.time - H.co2overloadtime > 300) // They've been in here 30s now, lets start to kill them for their own good!
 				H.adjustOxyLoss(8)
 		if(prob(20)) // Lets give them some chance to know somethings not right though I guess.
-			spawn(0) H.emote("cough")
-
+			spawn(0)
+				H.emote("cough")
+		//throw_alert("co2", /obj/screen/alert/too_much_co2) //we're gonna leave this one as a silent killer for now
```

Comment:
but it's useful commented out code, we might want to wreck co2 one day







nah we won't